k8s/helm-chart/kviklet/values.yaml

```diff
@@ -0,0 +1,35 @@
+#### image
+## kviklet Image
+image: ghcr.io/kviklet/kviklet:0.4.0
+
+#### imagePullPolicy
+## Pull Policy for Kubernetes Deployment
+imagePullPolicy: IfNotPresent
+
+#### pullSecret
+## Pull Secret for Docker Image
+imagePullSecret: dockerconfigjson
```

Comment:
what is the value here? I don't think this is defined.

Reply:
Another part can be removed